### DISCUSSION TITLE
Avoid alter table comment commands

### DISCUSSION THREAD

**Discussion by jsuarezgg** [dbt-labs/dbt-core#5645] [Q&A]
Hey everyone!
I’m trying to make some dbt models run faster, one of the main option is to avoid the
`alter table silver.table change column 
            name_column
            comment ""`
commands but I can’t figure out a way to do so, does someone in here have faced the same problem and solved it?

**Comment by jaypeedevlin** [comment]
Hey @jsuarezgg, do you need the comments persisted?  If not you can turn the feature off using the `persist_docs` config.

https://docs.getdbt.com/reference/resource-configs/persist_docs

**Reply by jsuarezgg** [reply]
Yes, I need the docs to stay on my databricks tables but I don't want dbt to do execute those alter table statements every time my incremental model runs

**Reply by datacom-bozhu** [reply]
@jaypeedevlin , I think what @jsuarezgg meant was when persist_docs is set to true for columns and there are some columns without descriptions in the model property yml file, then how can we get DBT not to run "alter table ... comment" statements for these columns to reduce total model running time.

As of DBT Core 1.6.6, it sends "alter table ... comment" statements for all columns which is wasting a reasonable amount of running time  and money when there are many models.
